ugs-core/src/com/willwinder/universalgcodesender/pendantui/PendantConfigBean.java

```diff
@@ -13,6 +14,9 @@
 		stepSizeList.add(new StepSizeOption("5", "5", false));
 		stepSizeList.add(new StepSizeOption("10", "10", true));
 		stepSizeList.add(new StepSizeOption("50", "50", false));
+
+		unitList.add(new UnitAmtOption("MM", "MM", true));
+		unitList.add(new UnitAmtOption("IN", "IN", false));
```

Comment:
For the value, you should be able to use `Units.MM.abbreviateion` and `Units.INCH.abbreviation`. That way you can use the deserialize helper later on.